themes/digital.gov/src/js/common/subscribe.js

```diff
@@ -0,0 +1,9 @@
+// scroll form into view when on /subscribe page
+const subscribeButton = document.querySelector(".subscribe-button");
+const signupForm = document.querySelector("#newsletter-signup ");
+const form = document.querySelector("#signup-form");
```

Comment:
Let's remove this form const if it's not being used any where.

Reply:
`form` is now removed.